keygen/src/keygen.rs

```diff
@@ -37,6 +37,18 @@ use {
     },
 };
 
+mod smallest_length_44_public_key {
+    use solana_sdk::declare_id;
+
+    declare_id!("21111111111111111111111111111111111111111111");
```

Comment:
```suggestion
    pubkey!("21111111111111111111111111111111111111111111");
```

more accurate as we're not talking about a program ID here